### DISCUSSION TITLE
Screen update flashes

### DISCUSSION THREAD

**Discussion by skinda0** [VasilKalchev/LiquidMenu#115] [Q&A]
Hi,
I have just been playing about with the menu library for a little project, and it is so much easier to use than other libraries! Managed to get exactly what I wanted within a few hours! I have one little query. I am using a rotary encoder to change values in the menu, but I have noticed when it updates the value, the LCD looks like it clears and redraws the screen causing it to 'flash'.

Is there something I can change to stop this happening?

**Comment by VasilKalchev** [comment]
Hi @skinda0, I'm glad you like the library.

You can try using the method [`LiquidMenu::softUpdate()`](https://vasilkalchev.github.io/LiquidMenu/class_liquid_menu.html#a506aa5f8822031096a881aed7c6e45e5), it won't clear the screen.
